packages/cli/src/cli/log.ts

```diff
@@ -0,0 +1,18 @@
+import { isArgo } from '@basemaps/cogify/build/argo.js';
+import { LogConfig, LogType } from '@basemaps/shared';
+import { CliInfo } from '@basemaps/shared/build/cli/info.js';
+import { flag } from 'cmd-ts';
+
+export const logArguments = {
+  verbose: flag({ long: 'verbose', description: 'Enable verbose logging' }),
+  extraVerbose: flag({ long: 'extra-verbose', description: 'Extra verbose logging' }),
+};
+
+export function getLogger(cli: { name: string }, args: { verbose: boolean; extraVerbose: boolean }): LogType {
```

Comment:
I am moving this into `@basemaps/shared` package in #3429 and reuse for all cli packages. But we can remove this later in the following tickets.